process/interceptors/processor/argTxInterceptorProcessor.go

```diff
@@ -0,0 +1,8 @@
+package processor
+
+import "github.com/ElrondNetwork/elrond-go/dataRetriever"
+
+// ArgTxInterceptorProcessor is the argument for the interceptor processor used for transaction-like data
+type ArgTxInterceptorProcessor struct {
```

Comment:
Agree. Might use Arg[something] as pattern because we read from left to right and after reading 3 chars we know we are reading an argument-type of struct

Reply:
done